Makefile

```diff
@@ -20,7 +20,7 @@ all: generate experimental
 # Check verifies that all of the committed files that are generated are
 # up-to-date.
 .PHONY: check
-check: generate experimental test fmt misspell makelint
+check: generate experimental test fmt misspell_warn makelint
```

Comment:
Since we are changing this, should we not also delete the original `misspell` task down below?

Reply:
I put the new task in under a different name so I could leave the original task as a standalone in case people already use that in their processes and want to continue doing so on platforms where it still works.

Reply:
Ah ok, that makes sense